setup.py

```diff
@@ -1,3 +1,4 @@
+import setuptools
 from setuptools import setup, find_packages, Extension
```

Comment:
I don't know if it should go before or after ?
Or maybe it's better to just `import setuptools` and adapt the rest? _(readability?)_